spec/spec_helper.rb

```diff
@@ -112,7 +112,10 @@ def only(*whitelist)
   end unless method_defined?(:only)
 end
 
-# set default time zone in TZ environment variable
-# which will be used to set session time zone
-ENV['TZ'] ||= 'Europe/Riga'
-# ENV['TZ'] ||= 'UTC'
+# Set Oracle Database session timezone
+# this setting needs to be in place to make sure that Oracle properly represents
+# the timestamp format on your local machine
+# Do not use 'OS_TZ' value or fixed offset values like '05:00', for daylight saving-enabled timezones
+# The setting cannot be derived directly from Windows operating system or ruby, as on the Windows machines,
+# timezone names are non-standard ones and are subject to translations to different languages.
+ENV['ORA_SDTZ'] ||= 'Europe/London'
```

Comment:
On my machine test were passing, when setting the `ORA_SDTZ='Europe/London'`
as my local timezone is actually "Europe/London".
Some test might have failed for you if you local timezone is different.
What I have noticed it that Ruby handles local timezone with DST more-less correct while Oracle uses fixed offset :(
That's why I've decided to fix only one side (Oracle), as setting timezone from Ruby will not do on Windows.
On windows tzutil /l gives following timezone names:
```
...
(UTC+10:00) Guam, Port Moresby
West Pacific Standard Time

(UTC+10:00) Hobart
Tasmania Standard Time

(UTC+10:00) Magadan
Magadan Standard Time

(UTC+10:00) Władywostok, Magadan (RTZ 9)
Vladivostok Standard Time
...
```
While Oracle valid timezone names `SELECT * FROM v$timezone_names` are like:
```
Africa/Algiers                                                   LMT
Africa/Algiers                                                   PMT
Africa/Algiers                                                   WET
Africa/Algiers                                                   WEST
Africa/Algiers                                                   CET
Africa/Algiers                                                   CEST
```